package.json

```diff
@@ -23,9 +23,9 @@
   "author": "The Actions Ecosystem Authors",
   "license": "Apache 2.0",
   "dependencies": {
-    "@actions/core": "^1.2.6",
-    "@actions/github": "^3.0.0",
-    "@octokit/rest": "^17.9.3",
+    "@actions/core": "^1.6.0",
+    "@actions/github": "^5.0.0",
+    "@octokit/rest": "^18.12.0",
```

Comment:
[Out of scope] It seems that @octokit/rest was actually unnecessary since before. However, this is out of scope for this pull request, so no action is needed.